### ISSUE TITLE
Azure OpenAI summarize_provider within class Groundedness throws error

### ISSUE THREAD

**Issue by ingridstevens** [truera/trulens#747]
On ["Initialize Feedback Function(s)" section of the Azure OpenAI example](https://github.com/truera/trulens/blob/1b7c5167032033448af74f35f43710c8884dddec/trulens_eval/examples/expositional/models/azure_openai.ipynb) the following code:

```
# groundedness of output on the context
groundedness = feedback.Groundedness(summarize_provider=azopenai, groundedness_provider=azopenai)
```
throws an error:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[29], [line 18](vscode-notebook-cell:?execution_count=29&line=18)
     [13](vscode-notebook-cell:?execution_count=29&line=13) f_qs_relevance = Feedback(azopenai.qs_relevance, name = "Context Relevance").on_input().on(
     [14](vscode-notebook-cell:?execution_count=29&line=14)     TruLlama.select_source_nodes().node.text
     [15](vscode-notebook-cell:?execution_count=29&line=15) ).aggregate(np.mean)
     [17](vscode-notebook-cell:?execution_count=29&line=17) # groundedness of output on the context
---> [18](vscode-notebook-cell:?execution_count=29&line=18) groundedness = feedback.Groundedness(groundedness_provider=azopenai, summarize_provider=azopenai)
     [19](vscode-notebook-cell:?execution_count=29&line=19) f_groundedness = Feedback(groundedness.groundedness_measure, name = "Groundedness").on(TruLlama.select_source_nodes().node.text).on_output()

TypeError: Groundedness.__init__() got an unexpected keyword argument 'summarize_provider'
```

I believe this is because in the Class definition for `Groundedness` the Args list  a `summarize_provider` but it doesn't seem to be used (that I could find at least) 

```
Args:
            groundedness_provider (Provider, optional): groundedness provider options: OpenAI LLM or HuggingFace NLI. Defaults to OpenAI().
            summarize_provider (Provider, optional): Internal Usage for DB serialization.
        """
```

My workaround was to eliminate the `summarize_provider` from the groundedness var definition: `groundedness = feedback.Groundedness(groundedness_provider=azopenai)`

----
Note on versions:
`trulens-eval                             0.20.0`
`openai                                   1.3.7`

**Comment by joshreini1** [comment]
Hey @ingridstevens  - this is the right solution. Summarize_provider was removed a few versions back.

**Comment by ingridstevens** [comment]
Thanks @joshreini1 for clarifying that the feature is no longer part of Trulens. I appreciate the information. However, I noticed that even though it has been removed, the example notebook for Azure still references to it.

I wanted to bring this to your attention, as it might cause confusion or errors for someone using the example Azure notebook at this link: https://github.com/truera/trulens/blob/main/trulens_eval/examples/expositional/models/azure_openai.ipynb
